meshnets/eval.py

```diff
@@ -0,0 +1,65 @@
+"""The main file for model evaluation."""
+
+import os
+
+from absl import app
+from absl import flags
+import pytorch_lightning as pl
+import torch
+from torch.utils.data import random_split
+from torch_geometric.loader import DataLoader
+
+from meshnets.utils.datasets import FromDiskGeometricDataset
+from meshnets.utils import model_loading
+
+FLAGS = flags.FLAGS
+
+# Random seed flag
```

Comment:
Remove these comments. The string description of the Flag must be explanation enough